packages/bump-monorepo-packages/src/bump-packages.js

```diff
@@ -5,7 +5,7 @@ const path = require('path');
 const childProcess = require('child_process');
 const assert = require('assert');
 const { promisify } = require('util');
-const getPackagesInTopologicalOrder = require('./get-packages-in-topological-order');
+const { getPackagesInTopologicalOrder } = require('@mongodb-js/monorepo-tools');
```

Comment:
NIT: do we want to move this package code inside the other one? this seems to be also a monorepo tool type of package

Reply:
That's what I asked in DMs and in the PR description.